web/admin.py

```
@@ -811,3 +817,94 @@ class PointsAdmin(admin.ModelAdmin):
+    search_fields = ('name', 'description', 'region')
+    prepopulated_fields = {'slug': ('name',)}
+    date_hierarchy = 'created_at'
+    readonly_fields = ('founding_date', 'created_at', 'updated_at')
+    fieldsets = (
+        (None, {'fields': ('name', 'slug', 'description', 'region', 'logo')}),
+        ('Status', {'fields': ('is_active', 'is_featured')}),
+        ('Links', {'fields': ('website', 'discord_link', 'facebook_link', 'twitter_link')}),
+        ('Dates', {'fields': ('founding_date', 'created_at', 'updated_at')}),
+    )
+
+    def member_count(self, obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `member_count`

(ANN201)